application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -121,10 +126,10 @@ public VcActivityCommand() {
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
         Member member = Objects.requireNonNull(event.getMember(), "member is null");
         GuildVoiceState voiceState = Objects.requireNonNull(member.getVoiceState(),
-                "Voicestates aren't being cached, check the JDABuilder");
+                "Voice states aren't being cached, check the JDABuilder");
```

Comment:
Not for this PR and disagree

Reply:
Imo mini fixes like typos can be done in another PR as well. Other than that I would agree though.